assets/javascripts/test_result.js

```diff
@@ -749,11 +749,37 @@ function renderInvestigationTab(response) {
         keyElement.style.verticalAlign = 'top';
         keyElement.appendChild(document.createTextNode(key));
 
+        var text_lines = response[key].split('\n');
```

Comment:
Please use the same style we commonly use in JavaScript code (textLines instead of text_lines).